.github/workflows/integration-opensuse-leap.yml

```diff
@@ -0,0 +1,37 @@
+name: integration-opensuse-leap-15
```

Comment:
Just small typo: shoudn't file name matching the workflow name? I mean, just for coherence as we do for the other unit workflow

Reply:
Fixed